### PR TITLE
Add PyPI deployment

### PR DESCRIPTION
**Pull request by gilesknap**

### CODE REVIEW

**Comment by gilesknap** [comment]
This demos that an un-tagged commit does not publish to PyPI

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -46,6 +46,13 @@ jobs:
           pip install coveralls
           coveralls
 
+      - name: Publish to PyPI
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && matrix.python-version == '3.7'
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
```
Comment (thomascobb) [inline .github/workflows/build.yml:54]: Can we make this secrets.pypi_token?

Comment (gilesknap) [reply .github/workflows/build.yml]: Yes that would be better. Only somehow it has broken it. Now I'm doubting that I got organisation secrets working. Continuing to investigate ...

Comment (gilesknap) [reply .github/workflows/build.yml]: Ah - the magic is don't but a carriage return at the end of your token

**Pull request merged**